themes/ssp/attrauthcomanage/noty.tpl.php

```diff
@@ -77,7 +77,9 @@
     <?php if (!empty($this->data['noty']['icon'])) : ?>
     <img class="gif-box" src="resources/icons/<?= $this->data['noty']['icon'] ?>">
     <?php endif; ?>
-    <p><?= $this->t('{themeegi:attrauthcomanage_noty:title}') ?></p>
+    <?php if(!empty($this->data['noty']['title'])): ?>
+    <p><?= $this->t('{attrauthcomanage:noty:' . $this->data['noty']['title'] . '}') ?></p>
```

Comment:
Is `title` renamed to `resend_confirmation_email`? If yes, this should be changed as well

Reply:
`resend_confirmation_email` is the key pulled from the dictionary that will be then found in the `title` key of the noty configuration